src/app/filter-gender/filter-gender.component.ts

```diff
@@ -0,0 +1,31 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-qpac-filter-gender',
+  templateUrl: './filter-gender.component.html',
+  styleUrls: ['./filter-gender.component.scss']
+})
+export class FilterGenderComponent implements OnInit {
+  gender = '';
```

Comment:
Good job!
I could give you an alternative way of solving this task using only one method and property:
```typescript
enum Gender {
  Male, // default value is 0
  Female, // default value is 1
}

selectedGender: Gender;

selectGender(gender) {
 this.selectedGender = gender;
}
```

Rest of implementation you can do on your own :)